src/main/java/core/Main.java

```diff
@@ -0,0 +1,60 @@
+package core;
+
+import core.controller.FruitShop;
+import core.model.Fruit;
+import core.model.TypeOperations;
+import core.service.oparation.BalanceHandler;
+import core.service.oparation.OperationHandler;
+import core.service.oparation.PurchaseHandler;
+import core.service.oparation.ReturnHandler;
+import core.service.oparation.SupplyHandler;
+import core.service.validator.ForScannerValidatorImpl;
+import core.service.writer.OperationWriter;
+import core.service.writer.OperationWriterImpl;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        writeReportToFile();
+        writeDailyReport();
+    }
+
+    private static void writeReportToFile() {
+        // Crate Map with operation handlers:
+        Map<String, OperationHandler> operationHandlersMap = new HashMap<>();
+        operationHandlersMap.put(TypeOperations.BALANCE.get(), new BalanceHandler());
+        operationHandlersMap.put(TypeOperations.SUPPLY.get(), new SupplyHandler());
+        operationHandlersMap.put(TypeOperations.PURCHASE.get(), new PurchaseHandler());
+        operationHandlersMap.put(TypeOperations.RETURN.get(), new ReturnHandler());
+        // Add fruits in StorageMap operation handlers:
+        FruitShop.STORAGE.getFruitStorageMap().put(new Fruit("banana"), 0);
+        FruitShop.STORAGE.getFruitStorageMap().put(new Fruit("apple"), 0);
+        // Run generate FruitShop report:
+        String inputFileCsv = "src/main/resources/inputFile.csv";
+        String reportFileCsv = "src/main/resources/reportFile.csv";
+        FruitShop fruitShop = new FruitShop(operationHandlersMap, inputFileCsv, reportFileCsv);
```

Comment:
У тебе FruitShop не має знати ні про які файли. Зчитуванням і записуванням у файл у нас займається FileReader i FileWriter.
```suggestion
        FruitShop fruitShop = new FruitShop(operationHandlersMap);
```

Reply:
I deleted it.